manifests/init.pp

```diff
@@ -4,10 +4,14 @@
 #
 # @see http://www.freedesktop.org/software/polkit/docs/latest/ PolicyKit Documentation
 #
+# @param package_ensure The ensure status of packages
+#
 # @author Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
 #
-class polkit {
+class polkit (
+  Polkit::PackageEnsure $package_ensure = 'installed'
```

Comment:
yea, having a simp_option to change package_ensure globally would be a good idea, as long as they are exposed as parameters to individual modules like this.